src/main/scala/wdl4s/wdl/util/StringUtil.scala

```diff
@@ -47,14 +47,14 @@ object StringUtil {
     @tailrec
```

Comment:
I suspect these could be folds, but since that condition isn't the one `wdl4s` is going under the knife for I'll just leave that TOL there